src/main/resources/default/assets/common/core.js.pasta

```diff
@@ -691,3 +691,64 @@ sirius.extractFileExtension = function (filename) {
     }
     return '.' + nameAndExtension.pop();
 }
+
+/**@
```

Comment:
What does this solve that https://github.com/scireum/sirius-web/blob/develop/src/main/resources/default/assets/tycho/scripts/clipboard.js.pasta can't already do (except the scoping)?

Reply:
Just my opinion:
* Better scoping (along the other helper methods)
* More robust fake element / fallback handling
* Tried and tested code that ran a long time in OX (I can't find a call to the function from clipboard js)
* Marking the element with a class (which is a required use case for OX